recipes/tarts/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "tarts" %}
+{% set version = "0.0.0a4" %}  # Update this to your current version
+{% set python_min = "3.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+# 
+source:
+  url: https://github.com/PetchMa/TARTS/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 44189755bd476d5de1c65a6dc1748454661bc6c9c93adf371f391cf197b85f9b
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation"
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - pip
+    - setuptools
+  run:
+    - python >={{ python_min }}
+    - numpy
+    - scipy
+    - matplotlib
+    - pytorch
+    - pytorch-lightning
+    - pyyaml
+    - torchvision
+    - astropy
+    - pandas
+    - gitpython
+
+test:
+  requires:
+    - python {{ python_min }}
+  imports:
+    - TARTS
+  commands:
+    - python -c "import TARTS; print('Package imported successfully')"
```

Comment:
These two commands are not necessary. The first is redundant with above `imports` test. The second also looks like you would get from the package's `__init__.py`. You could probably add to `imports`, `TARTS.NeuralActivityOpticsSys` if necessary.